src/test/java/Homework18.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+
+    @Test
+    public void playSong() {
+
+        castEmail("dirzo@gmail.com");
+        castPassword("Te$ter1234");
+        castSubmit();
+        castPlay();
+        Assert.assertTrue(summonSong());
+    }
```

Comment:
Hi,
I have checked your homework 18, but suite HW18 run failed. Please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/44592718/24589e13-f118-4405-8ba4-3042ca4fdb58)

In this error above, failed because no such element `selector":"input[type='email'"`. In your currently Homework 18 code, I don't see you add step open koel link.
//HINT: add step `driver.get(url);` come in first step `castEmail("dirzo@gmail.com");`

Reply:
Fixed